cli/command/container/utils_test.go

```diff
@@ -60,10 +60,13 @@ func TestWaitExitOrRemoved(t *testing.T) {
 		},
 	}
 
-	client := &fakeClient{waitFunc: waitFn, Version: api.DefaultVersion}
-	for _, testcase := range testcases {
-		statusC := waitExitOrRemoved(context.Background(), client, testcase.cid, true)
-		exitCode := <-statusC
-		assert.Check(t, is.Equal(testcase.exitCode, exitCode))
+	for _, tc := range tests {
+		t.Run(tc.cid, func(t *testing.T) {
+			client := &fakeClient{waitFunc: waitFn, Version: api.DefaultVersion}
+
```

Comment:
It's now using a separate client per test – was that an intentional change?

Reply:
Good catch; not _really_ (I think I contemplated "parallel" tests for a second, but the overhead thereof didn't make much sense for this).

Let me change that back

Reply:
@laurazard updated; PTAL